src/components/UserSettingsModal/UserSettingsModal.jsx

```diff
@@ -1,5 +1,12 @@
+import UserSettingsForm from 'components/UserSettingsForm/UserSettingsForm';
+import css from './UserSettingsModal.module.css';
 const UserSettingsModal = () => {
-  return <div>UserSettingsModal</div>;
+  return (
+    <div className={css.wrapper}>
+      <h1 className={css.title}>Setting</h1>
```

Comment:
please add this attribute  id="modalTitle"
also please check if h1 it is normal to modal title